comp/core/workloadmeta/collectors/util/ecs_util.go

```diff
@@ -0,0 +1,267 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build docker
+
+package util
```

Comment:
base on this PR  I'm not sure we need an util package. Both `TaskParser` and `IsTaskCollectionEnabled` is only used on `comp/core/workloadmeta/collectors/internal/ecs/ecs.go`.

Base on that and the comments below we can probably add the function we have here to the packages they are using it

Reply:
Actually I have another PR to update ECS fargate collector, both collectors use `TaskParser` and `IsTaskCollectionEnabled`, so I created `ecs_util.go `
https://github.com/DataDog/datadog-agent/pull/23253/files#diff-b755149238dd28f3c616d3974000977c004ebfb915be4340ce601f868a43420bR37